razzle.config.js

```diff
@@ -1,4 +1,26 @@
 module.exports = {
+  entry: ['babel-polyfill', './src/client.js'],
```

Comment:
You will want to move these additions into the `if(!dev)` block below, to run for the production build. 
There you can push preset values to `appConfig.presets` and the entry to `appConfig.entry`, since we want to add config, not overwrite it completely.